### PR TITLE
Add spider chi_ssa_64

### PR DESCRIPTION
**Pull request by DiegoCaraballo**
## Summary

**Issue:** #909 

## Checklist

All checks are run in [GitHub Actions](https://github.com/features/actions). You'll be able to see the results of the checks at the bottom of the pull request page after it's been opened, and you can click on any of the specific checks listed to see the output of each step and debug failures.

- [ ] Tests are implemented
- [ ] All tests are passing
- [ ] Style checks run (see [documentation](https://cityscrapers.org/docs/development/) for more details)
- [ ] Style checks are passing
- [ ] Code comments from template removed

## Questions

Include any questions you have about what you're working on.

### CODE REVIEW

File: city_scrapers/spiders/chi_ssa_64.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+from datetime import datetime
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa64Spider(CityScrapersSpider):
+    name = "chi_ssa_64"
+    agency = "Chicago Special Service Area #20 Walden Parkway"
```
Comment (pjsier) [inline city_scrapers/spiders/chi_ssa_64.py:12]: This should be 64 instead of 20

File: city_scrapers/spiders/chi_ssa_64.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    timezone = "America/Chicago"
+    start_urls = ["https://www.mpbhba.org/business-resources/"]
+
+    def parse(self, response):
+        meetings = self._parse_block(response)
+        for item in meetings:
+            meeting = Meeting(
+                title='Morgan Park Commission',
```
Comment (pjsier) [inline city_scrapers/spiders/chi_ssa_64.py:20]: "Commission" is fine here

File: city_scrapers/spiders/chi_ssa_64.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        meetings = []
+        count = 0
+        general_block = response.xpath(
+            '//*[@class="et_pb_module et_pb_text et_pb_text_2 '
+            'et_pb_text_align_left et_pb_bg_layout_light"]'
+        )
+        years = general_block.xpath('.//*[@class="et_pb_text_inner"]/h3/text()').extract()
+        specific_block = general_block.xpath('.//*[@class="et_pb_text_inner"]'
+                                             ).extract_first().split("<strong>")
+        for line in specific_block:
```
Comment (pjsier) [inline city_scrapers/spiders/chi_ssa_64.py:50]: This was my mistake in the initial issue, but we only want items for SSA 64 here

File: city_scrapers/spiders/chi_ssa_64.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        date_string = (
+            item[0] + result[1] + " " + time_zone + " " +
+            item[1].split("a.m.")[0].split(",")[2].strip()
+        )
+        start = datetime.strptime(date_string, "%Y %B %d %p %H")
+
+        return start
+
+    def _parse_location(self, item):
+        """Parse or generate location."""
```
Comment (pjsier) [inline city_scrapers/spiders/chi_ssa_64.py:81]: We'll want to include "Chicago, IL" at the end if it isn't present

**Comment by DiegoCaraballo** [comment]
@pjsier Ok, I will make the corresponding changes.
Regards

File: city_scrapers/spiders/chi_ssa_64.py [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+        result = (
+            item[1].strip() + " Chicago, IL" if
+            ("Chicago, IL" not in item[1].strip()) else item[1].strip()
+        )
+        result = re.split('a.m.|p.m.', result)[1]
+
+        return {"address": result.split(",")[1].strip(), "name": result.split(",")[0].strip()}
```
Comment (pjsier) [inline city_scrapers/spiders/chi_ssa_64.py:88]: Looks like this piece is dropping the ", IL" from "Chicago, IL". You might be able to use something like this:

```python
','.join(result.split(',')[1:])
```

Comment (DiegoCaraballo) [reply city_scrapers/spiders/chi_ssa_64.py:88]: Thanks @pjsier 
This is correct. I did not see it.
In the new version it will be resolved.

**Pull request merged**